### PR TITLE
Create a handbook page for program management

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: operations/program-management/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+* Hanna Park: Program Manager
+* Vidhi Saini: Program Coordinator
+
+## Programs
+
+The team manages several cross-functional programs, including:
+
+1. Non-MME Customer Operations
+
+ - (Vidhi) Non-commercial account management: All non-commercial renewals executed through Program Management, including non-profit, gratis and a subset of academic licenses.
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
 - (Vidhi) Non-commercial account management: All non-commercial renewals executed through Program Management, including non-profit, gratis, and a subset of academic licenses.
```

File: operations/program-management/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - Non-profit process: https://docs.google.com/document/d/1ey7bIXzuk71kyPyPqDpN2p-OCTTJmcKs9AzB-0q3f6c/edit
```
Comment (cwarnermm) [inline operations/program-management/README.md:25]: FYI: Gdoc currently isn't accessible with Mattermost staff.

File: operations/program-management/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ - (Owner TBD) Self-service operations: All commercial non-MME renewals with <$5K ARR executed through Program Management, via self-serve.
+    - *Process to be added*
+ - (Owner TBD) Out-of-band non-MME requests: Managing requests, updating docs, routing, and policies to increase efficiency and speed.
+    - *Process to be added*
+ - (Owner TBD) Manual license key provisioning and administration: License key provisioning and tracking, outside of automated sales ops workflows, including manual sales-serve Cloud subscriptions & Cloud trial extensions.  
+    - *Process to be added*
+ - (Hanna) Channel partner operations: Management of gratis partner license key program, maintaining and adding partner marketplace listings, channel operations docs, fulfillment, partner agreements
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
 - (Hanna) Channel partner operations: Management of gratis partner license key program, maintaining and adding partner marketplace listings, channel operations docs, fulfillment, and partner agreements.
```

File: operations/program-management/README.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    - Partners/resellers list: https://docs.google.com/spreadsheets/d/1peej--v7WhjU58J_Wv-OW_iutT_MqqsTMMZeBS5qIBc/edit#gid=1855980138
+    - Partner programs: https://handbook.mattermost.com/operations/sales/partner-programs
+    - Partner directory: https://mattermost.com/partners/
+ - (Jason) US Fed SBIR program management: US Fed SBIR program management: Management of SBIR vendor relationship, documentation & submission of SBIR contracts.
+    - *Process to be added*
+
+2. Company Operations:
+ - (Vidhi) Operational scorecard: Managing metrics and reporting for cross-company scorecards
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
 - (Vidhi) Operational scorecard: Managing metrics and reporting for cross-company scorecards.
```

File: operations/program-management/README.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      - Scorecard: https://docs.google.com/spreadsheets/d/19TALKW9F7-BMm_bTLmP2nVLoG-RPbuvfdEk0ObSXD74/edit#gid=738758018
+ - (Jason) GNN reviews: Leading cross-company review and GNN read-out process
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
 - (Jason) GNN reviews: Leading cross-company reviews and GNN read-out process.
```

File: operations/program-management/README.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+      - *Process to be added*
+ - (Hanna) Onboarding/Offboarding: Attend onboarding calls, provide historical context for new people joining the team, answer questions.
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
 - (Hanna) Onboarding/Offboarding: Attend onboarding calls, provide historical context for new people joining the team, and answer questions.
```

File: operations/program-management/README.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      - *Process to be added*
+ - (Vidhi) Diversity, equity & inclusion (DE&I): Promoting DE&I within Mattermost staff and community, creating a welcoming environment for all.
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
 - (Vidhi) Diversity, equity, & inclusion (DE&I): Promoting DE&I within Mattermost staff and community, creating a welcoming environment for all.
```

File: operations/program-management/README.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+      - *Process to be added*
+
+3. Community Operations: 
+
+ - (John) Forum management: Monitoring and maintenance of peer-to-peer feature request forum on Uservoice and Discourse forum.
```
Comment (cwarnermm) [inline operations/program-management/README.md:51]: FYI: John isn't listed as a team member at the top of this page.

File: operations/program-management/README.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+ - (Hanna) Channel partner operations: Management of gratis partner license key program, maintaining and adding partner marketplace listings, channel operations docs, fulfillment, partner agreements
+    - Partners/resellers list: https://docs.google.com/spreadsheets/d/1peej--v7WhjU58J_Wv-OW_iutT_MqqsTMMZeBS5qIBc/edit#gid=1855980138
+    - Partner programs: https://handbook.mattermost.com/operations/sales/partner-programs
+    - Partner directory: https://mattermost.com/partners/
+ - (Jason) US Fed SBIR program management: US Fed SBIR program management: Management of SBIR vendor relationship, documentation & submission of SBIR contracts.
+    - *Process to be added*
+
+2. Company Operations:
+ - (Vidhi) Operational scorecard: Managing metrics and reporting for cross-company scorecards
+      - Scorecard: https://docs.google.com/spreadsheets/d/19TALKW9F7-BMm_bTLmP2nVLoG-RPbuvfdEk0ObSXD74/edit#gid=738758018
```
Comment (cwarnermm) [inline operations/program-management/README.md]: FYI: GDoc not accessible by Mattermost staff.

File: operations/program-management/README.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+ - (Vidhi) Diversity, equity & inclusion (DE&I): Promoting DE&I within Mattermost staff and community, creating a welcoming environment for all.
+      - *Process to be added*
+
+3. Community Operations: 
+
+ - (John) Forum management: Monitoring and maintenance of peer-to-peer feature request forum on Uservoice and Discourse forum.
+     - Discourse: https://forum.mattermost.com/
+     - UserVoice: https://mattermost.uservoice.com/forums/306457-general
+     - Discord: https://discord.gg/kAC8WakMAx (for OpenOps & AI communities)
+ - (Vidhi, with Hanna) Community promotions: Managing logistics around community swag, Mattermost contributor mugs, Mattermost contributor MVPs.
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
 - (Vidhi, with Hanna) Community promotions: Managing logistics around community swag, Mattermost contributor mugs, and Mattermost contributor MVPs.
```

File: operations/program-management/README.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+     - Community swag orders: https://docs.google.com/spreadsheets/d/1peej--v7WhjU58J_Wv-OW_iutT_MqqsTMMZeBS5qIBc/edit#gid=0
+ - (Hanna) Community event logistics: Managing logistics around Community/DevRel meetups-in-a-box, DevRel/engineering talks
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
 - (Hanna) Community event logistics: Managing logistics around Community/DevRel meetups-in-a-box, and DevRel/engineering talks.
```

File: operations/program-management/README.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
+# Program management
+
+**Program Management** leads and executes key cross-functional programs, working with cross-functional, cross-departmental stakeholders to **increase operational efficiency**.
```
Comment (cwarnermm) [inline operations/program-management/README.md]:
```suggestion
**Program Management** leads and executes key cross-functional programs, working with cross-functional and cross-departmental stakeholders to **increase operational efficiency**.
```

**Review by cwarnermm — DISMISSED** [review]
Feedback provided inline. Approving for expediency.

**Comment by jasonblais** [comment]
Thanks @cwarnermm! Updates addressed. Appreciate the swift review :)

**Pull request merged**